src/functional_data.Rmd

```diff
@@ -0,0 +1,245 @@
+
+```{r}
+# feat.methods specifies what to extract from which functional
+# In this example, from the first functional we extract the fourier transformation, from the second the fpca scores.
+feat.methods = list("UVVIS" = extractFDAFourier(), "NIR" = extractFDAFPCA())
+
+# Either create a new task from an existing task
+extracted = extractFDAFeatures(tsk1, feat.methods = feat.methods)
+extracted
+```
+
+
+### Wavelets
+
+In this case, discrete wavelet feature transformation is applied.
+Discrete wavelet transform decomposes the functional into several _wavelets_.
+This essentially transforms the time signal to a time-scale representation, where every wavelet has a different resolution.
+We can specify which additional parameters (i.e. the filter (type of wavelet) and the boundary) in the pars argument.
+This functions returns a regression task of type regr since the raw data contained temporal structure but the transformed data does not inherit temporal structure anymore.
+For more informations on wavelets consider the documentation [wavelets](dwt). A more comprehensive guide is for example given [here](https://www.cs.unm.edu/~williams/cs530/arfgtw.pdf).
+
+```{r, eval = FALSE}
+## Specify the feature extraction method and generate new task.
+## Here, we use the Haar filter:
+feat.methods = list("UVVIS" = extractFDAWavelets(filter = "haar"))
+task.w = extractFDAFeatures(tsk1, feat.methods = feat.methods)
+
+# Use the Daubechie wavelet with filter length 4.
+feat.methods = list("NIR" = extractFDAWavelets(filter = "d4"))
+task.wd4 = extractFDAFeatures(tsk1, feat.methods = feat.methods)
+```
+
+
+### Fourier transformation
+
+Now, we use the fourier feature transformation.
+The Fourier transform takes a functional and transforms it to a frequency domain by splitting the signal up into its different frequency components. A more detailed tutorial on fourier transform can be found [here](http://www0.cs.ucl.ac.uk/teaching/GZ05/03-fourier.pdf).
+Either the amplitude or the phase of the complex fourier coefficients can be used for analysis.
+This can be specified in the additional _trafo.coeff_ argument:
+
+```{r, eval = FALSE}
+# Specify the feature extraction method and generate new task.
+# We use the fourier features and the amplitude for NIR, as well as the phase for UVVIS
+feat.methods = list("NIR" = extractFDAFourier(trafo.coeff = "amplitude"),
+                    "UVVIS" = extractFDAFourier(trafo.coeff = "phase"))
+task.fourier = extractFDAFeatures(tsk1, feat.methods = feat.methods)
+task.fourier
+```
+
+### Wrappers
+Additionally we can wrap the preprocessing around a standard learner such as __classif.rpart__.
+For additional information, please consider the __Wrappers__ section.
```

Comment:
Which section do you mean? If you want to refer to the tutorial page about wrappers please use a link.